.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
+public
+secrets.env
```

Comment:
`secrets.env` contains docker-compose secrets. I would like to use this pattern in all our projects where we use docker-compose